rpm/xrootd-lotman.spec

```
@@ -35,6 +35,9 @@ This package provides a purge plugin for XRootD that uses Lotman's tracking for
 %{_includedir}/XrdPurgeLotMan.hh
 
 %changelog
+* Wed Apr 16 2025 MÃ¡tyÃ¡s Selmeci <mselmeci@wisc.edu> - 0.0.2-2
```

```suggestion
* Wed Apr 16 2025 Matyas Selmeci <mselmeci@wisc.edu> - 0.0.2-2
```
I think encoding issues messed up my name. Let's just go with plain ASCII for now